tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1555,8 +1556,8 @@ static SourceFile *retrieveInputFile(StringRef inputBufferName,
     // done that)
     if (haveRealPath)
       return nullptr;
-    std::string realPath =
-        SwiftLangSupport::resolvePathSymlinks(inputBufferName);
+    std::string realPath = resolveSymbolicLinks(inputBufferName,
+        llvm::vfs::getRealFileSystem().get());
```

Comment:
Same question here

Reply:
I updated the PR to use the ambient FileSystem.